lego/apps/followers/models.py

```diff
@@ -25,6 +25,7 @@ class FollowEvent(Follower):
     target = models.ForeignKey(
         "events.Event", related_name="followers", on_delete=models.CASCADE
     )
+    sent = models.BooleanField(default=False)
```

Comment:
Maybe rename this to something more understandable as well. f.ex `registration_notification_sent` or something shorter :stuck_out_tongue_closed_eyes: